.github/workflows/release.yml

```diff
@@ -139,6 +145,11 @@ jobs:
           graalvm-version: ${{ env.graalVersion }}
           java-version: ${{ env.javaVersion }}
           native-image: true
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ env.rustToolchain }}
+          override: true
```

Comment:
Maybe we could add an environment variable to skip the Rust check?

Currently we need to install the Rust toolchain when building the launcher solely to satisfy the check, as launcher has no Rust dependencies. Is it planned to make the launcher use Rust dependencies in the near future? If not maybe it's worth making it possible to build the launcher without setting up Rust?

Reply:
According to @iamrecursion the whole project is going to depend on rust in the end.

Reply:
When saying that I didn't consider that the launcher is truly separate. Perhaps that means that splitting up the version check is more sensible. Have it _warn_ at SBT startup when things are missing, and have that warning become an error when you try and compile a project that depends on Rust.